sdk/packages/eternum/src/types/index.ts

```diff
@@ -3,6 +3,13 @@ import { Account, num } from "starknet";
 interface SystemSigner {
   signer: Account;
 }
+export interface MintCC extends SystemSigner {
```

Comment:
this doesn't need to exist - just use SystemSigner

Reply:
np